x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/plan/logical/command/sys/SysColumnsTests.java

```diff
@@ -130,6 +138,35 @@ public void testInNonDriverMode() {
         }
     }
 
+    public void testUnsignedLongFiltering() {
+        List<SqlVersion> versions = List.of(
+            SqlVersion.fromId(INTRODUCING_UNSIGNED_LONG.id - SqlVersion.MINOR_MULTIPLIER),
+            INTRODUCING_UNSIGNED_LONG,
+            SqlVersion.fromId(INTRODUCING_UNSIGNED_LONG.id + SqlVersion.MINOR_MULTIPLIER),
+            SqlVersion.fromId(Version.CURRENT.id)
+        );
```

Comment:
This list is the same as https://github.com/elastic/elasticsearch/pull/65145/files#diff-8f9e4dcb53d1a3bb501882f5c67dfb19be7a0606fd27c2762060721d8d7fe2f3R108. Worth extracting it to a common place?

Reply:
Makes sense, extracted and reused.